content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -16,17 +16,276 @@ further_reading:
       text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
-W3C Trace Context propagation is available for all supported languages. The default trace propagation style for all languages is `datadog,tracecontext`. Envoy and nginx proxies use the default of `tracecontext,datadog`.
+Trace Context propagation is the mechanism of passing tracing information like Trace ID, Span ID, and sampling decisions from one part of a distributed application to another. This enables all traces (and additional telemetry) in a request to be correlated. When automatic instrumentation is enabled, trace context propagation is handled automatically by the tracing library.
 
-For more information about each language's configuration options for trace context propagation, see the following pages:
+By default, the Datadog tracing libraries read and write distributed tracing headers using both the **Datadog** format and the [**W3C Trace Context**][2] formats. When reading incoming headers, the language SDK's will giver higher precedence to the Datadog format, whereas Envoy and NGINX proxies will give higher precedence to the W3C Trace Context format.
 
-{{< partial name="apm/apm-context-propagation" >}}
+## Configuration
 
+If you need to customize the trace context propagation configuration, there are several environment variables you can use to configure the formats that are used for reading and writing distributed tracing headers. To enable a specific propagator, make sure to use the corresponding configuration value for the tracing library, as outlined in the **Language support** section.
```

Comment:
We should link to the language support section

Reply:
Added a link in aece8fb4f4281d29712db84828c2858300ef7e48 👍🏼